src/cryptography/x509.py

```diff
@@ -29,6 +31,77 @@ def __init__(self, msg, parsed_version):
         self.parsed_version = parsed_version
 
 
+class UnknownAttribute(Exception):
+    pass
+
+
+class Attribute(object):
```

Comment:
http://tools.ietf.org/html/rfc5280#section-4.1.2.4 is where this is from. The RFC calls it `AttributeTypeAndValue`. Suggestion? I don't have a better name than `Attribute` right now.